governator-api/src/main/java/com/netflix/governator/event/ApplicationEvent.java

```diff
@@ -0,0 +1,10 @@
+package com.netflix.governator.event;
+
+/**
+ * Marker interface for application events to be published
+ * to subscribed consumers.
```

Comment:
To make the javadoc more complete, reference classes with examples for publishing and consuming events.